tests/server/environment.spec.js

```diff
@@ -1,14 +1,40 @@
 const assert = require('assert')
+const path = require('path')
+const fs = require('fs/promises')
 
-const underTest = '../../src/server/environment'
+const env = require('../../src/server/environment')
 
 describe('environment settings', function () {
-  beforeEach('clear env module', function () {
-    delete require.cache[require.resolve(underTest)]
+  it('SERVER_CODE_DIR point to the right place', async function () {
+    const testPaths = [
+      'environment.js',
+      'lib/ensure-async-context.js'
+    ]
+
+    for (const relative of testPaths) {
+      const absolute = path.join(env.SERVER_CODE_DIR, relative)
+      const pathExists = await exists(absolute)
+      assert.ok(pathExists, `expected ${absolute} to exist, is SERVER_CODE_DIR wrong?`)
+    }
   })
 
-  it('points the CODE_DIR at src', function () {
-    const env = require(underTest)
```

Comment:
Oh, I'd guess this was to support updating env values while running tests in watch mode without restarting the script.

I've never tried to do that, so this seems fine to me.